src/ApiGenerator/RestSpecification/Core/delete_by_query.json

```diff
@@ -178,9 +178,9 @@
         "description": "The throttle for this request in sub-requests per second. -1 means no throttle."
       },
       "slices": {
```

Comment:
BWC issue?

Reply:
517142a deals with this, we ignore the fact this is a union for now